arctic/store/version_store.py

```diff
@@ -630,6 +822,21 @@ def _prune_previous_versions(self, symbol, keep_mins=120):
         cleanup(self._arctic_lib, symbol, version_ids)
 
     @mongo_retry
+    def delete_metadata(self, symbol):
```

Comment:
should we consider including an option to delete from versions too ?
Is the case where we delete only the timed meta the dominant one ?